doc/development/pullrequests.rst

```diff
@@ -125,13 +125,21 @@ the ``my_branch`` branch of meshmode.
 Reviewing/CI
 ------------
 
-Each pull requests for mirgecom needs one manual approval by a reviewer and
+Each pull request for mirgecom needs one manual approval by a reviewer and
 needs to pass the Continuous Integration (CI) tests before merging. For the
 manual reviews, please select at least one reviewer (someone that has
 knowledge about the code you are modifying) in the "Reviewers" box at the top
 right of a PR. You can set the PR as a "draft" PR to indicate that it is still
 in progress and only a high-level review is requested.
 
+.. note::
+
+   Resources about reviewing pull requests can be found here:
+
+   - `How to Do Code Reviews Like a Human (Part One) <https://mtlynch.io/human-code-reviews-1/>`__
+   - `How to Do Code Reviews Like a Human (Part Two) <https://mtlynch.io/human-code-reviews-2/>`__
+   - `How to Make Your Code Reviewer Fall in Love with You <https://mtlynch.io/code-review-love/>`__
+
```

Comment:
```suggestion
   Some thoughts and best practices regarding submitting your code for review can be found in this
   article:

   - `How to Make Your Code Reviewer Fall in Love with You <https://mtlynch.io/code-review-love/>`__
   
   A similar (but mirrored) set of concerns applies from the other direction. You will definitely
   want to read these articles when you start reviewing other folks' code, but it may be
   helpful to read them even before then to gain a better understanding of the process:

   - `How to Do Code Reviews Like a Human (Part One) <https://mtlynch.io/human-code-reviews-1/>`__
   - `How to Do Code Reviews Like a Human (Part Two) <https://mtlynch.io/human-code-reviews-2/>`__

```